### ISSUE TITLE
FiraCode Variants

### ISSUE THREAD

**Issue by john-f-chamberlain** [tonsky/FiraCode#1033]
Not so much an issue, but if anyone's using an editor that doesn't allow for stylistic sets and doesn't want to have to download python and customize the fonts yourself, I've uploaded every variation of the font available here:

https://f001.backblazeb2.com/file/FiraCode-Variants/index.html

**Comment by not-a-duck** [comment]
When installing the font, it has a really weird name like "Fira Code ss02 ss03 ss05 ss07"

**Comment by RubenVerg** [comment]
that's to distinguish from the main one

**Comment by RubenVerg** [comment]
my problem is, it looks very blurry

**Comment by Zerobim** [comment]
Could you add the result of the compilation to a separate file, or each zip? 
**Why?** 
There is currently an issue (#869) open about ss02 not being able to be baked in, and the python utility seems to issue a warning:
`WARNING: [applySubstitutions] Cannot remap 'greater_equal.liga' -> 'greater_equal.ss02' because neither has a Unicode value assigned in any of the cmap tables.
WARNING: [applySubstitutions] Cannot remap 'less_equal.liga' -> 'less_equal.ss02' because neither has a Unicode value assigned in any of the cmap tables.`

I tried with `FiraCode-ss02-ss03-ss05-ss06-ss07` and `ss02` does not seem to have been enabled.
I had to install the python utility (`pyftfeatfreeze`) to check if there was an issue with the zip file or with the utility.

Also if this gets automated, please consider adding the FiraCode release number or commit id.
The release or commit id of the utility would also help debug possible issues.

**Comment by john-f-chamberlain** [comment]
@not-a-duck 
> When installing the font, it has a really weird name like "Fira Code ss02 ss03 ss05 ss07"

Yes, this suffix is there to differentiate between the different versions of the font one might choose to download. I don't really have a cleaner way to do this at the moment without adding extra code. The pyftfeatfreeze utility automatically adds that suffix for me by adding a simple flag.

@Zerobim 
> Could you add the result of the compilation to a separate file, or each zip?
> Or at least mention warnings or errors during compilation. (<- Added on edit)

This is a good idea and one that I think I could add relatively easily. I'll look into what changes would be needed for this to happen.

**Comment by wundrweapon** [comment]
I am not using this currently but thank you for creating this!!

**Comment by IvanShpotenko** [comment]
`ss07` does not work for me either, tried pre-baked fonts from here https://f001.backblazeb2.com/file/FiraCode-Variants/index.html and also tried to bake myself with [OTFeatureFreeezer GUI](https://github.com/twardoch/fonttools-opentype-feature-freezer)
